src/asciidoctor-core-api.js

```diff
@@ -1465,7 +1465,61 @@ Cursor.prototype.getLineNumber = function () {
 };
 
 // Logger API (available in Asciidoctor 1.5.7+)
-// REMIND: we are using "skip_missing" because this API is only available starting with Asciidoctor 1.5.7
+
+function initializeLoggerFormatterClass (className, functions) {
```

Comment:
This code is mostly copied-pasted from `initializeProcessorClass`.
I think we should create a base function.